src/library_gl.js

```diff
@@ -7503,7 +7501,6 @@ var LibraryGL = {
     // TODO: This should be a trivial pass-though function, but due to https://bugzilla.mozilla.org/show_bug.cgi?id=1202427,
```

Comment:
overall lgtm. about this line: that bug has been closed a while ago, can we stop ignoring the range then?

Reply:
Hmm yeah indeed.. Let me merge as is for now though, I think ideally we'd handle these kinds of scenarios with a

```js
#if MINIMUM_FIREFOX_VERSION >= 51
GLctx.drawRangeElements(mode, start, end, count, type, indices);
#else
_glDrawElements(mode, count, type, indices);
#endif
```

then we could keep the fallback through the end of times, and additionally it will give a way for people to easily flip between these, e.g. to see if there's a performance differential (whenever it comes to WebGL, every entry point is a suspect)